src/tenant/views.py

```diff
@@ -57,14 +59,12 @@ class TenantCreate(PublicOnlyViewMixin, LoginRequiredMixin, CreateView):
 
     def form_valid(self, form):
         """ Copy the tenant name to the schema_name and the domain_url fields."""
-        from siteconfig.models import SiteConfig
-
         # TODO: this is duplication of code in admin.py.  Move this into the Tenant model?  Perhaps as a pre-save hook?
         form.instance.schema_name = form.instance.name.replace('-', '_')
         form.instance.domain_url = f'{form.instance.name}.{Site.objects.get(id=1).domain}'
 
         # save the form and get the response (HttpResponseRedirect)
-        response = super().form_valid(form)
+        self.object = form.save()
```

Comment:
Is there a reason to skip the super().form_valid() here?  That seems dangerous, as we're only checking some extra things here, not EVERYthing?  I'm not sure, but please clarify why it's not needed if you think it is not needed.

It may be fine as you have it. Please just explain as I'm not sure. THanks!

Reply:
I think I "unpacked" form_valid method, when I tried to find the source of the bug and left it as-is. Note that `CreateView` class inheritted `form_valid` method from `ModelFormMixin` class:

![Screenshot 2023-11-26 at 11 50 01](https://github.com/bytedeck/bytedeck/assets/144783/2f5b52cc-2766-4a44-964b-7bc80c031048)


which inherit it from `FormMixin`:

![Screenshot 2023-11-26 at 11 50 08](https://github.com/bytedeck/bytedeck/assets/144783/c901505c-adc4-470f-bbf2-aa2cc189cdf7)


so not a big deal, but I reverted this unnecessarry changes, that may (but I doubt it) be a source of problems in a future.

Reply:
Thanks for the explanation and update!